### ISSUE TITLE
Generate TypeScript context Azure Pipelines

### ISSUE THREAD

**Issue by ferodom** [delegateas/Daxif#45]
**Describe the bug**
I am using the YAML Pipelines to build my CRM system. I am using the nuget task to restore the packages. But that does not seem to update the dependencies or add the daxIf/bin folder needed or the other.fsx scripts. How are you guys generating the typescript context?
Are you checking in all the files on the DaxIf project?

**Environment**
- D365/CDS version: 9.1.x
- Tool version: Laters

**Additional context**
Add any other context about the problem here. Error messages og solution files.

**Comment by magesoe** [comment]
Looks like the daxif bin directory is in version control on my projects. I seem to remember @bo-stig-christensen telling me why at some point. Perhaps he can clarify here

**Comment by bo-stig-christensen** [comment]
@ferodom : I was trying to do the exact same as you, have Daxif (and the other Tooling components)  installed without having the binary directories incl. the binaries committed to Git. - As we know, it is best practice to avoid these binaries, or at the very least have them in Git LFS.

Having the bins included in the reporitory, ensures that you basically just boot up the solution and all is working.
However, upon removing the binaries, it was not working. Not even after Nuget Restore. - What had to be done was to run the following in Package Manager: "Update-Package -reinstall". - Without that, I could not make Nuget pick up the binaries correctly.

In our [XrmFramework](https://github.com/delegateas/XrmFramework) repo, I then rolled back to having the binaries in the repo until being able to figure out he root cause and a more viable solution.

Basically, we want to make sure that you are able to clone the XrmFramework repository and start a new project without adding too much manual work, including Nuget Reinstall, if possible.

I/We have a pending task to give XrmFramework a major overhaul from A to Z, but available time is usually the showstopper. - Looking at my time in June and July, it does however look good, so I am hoping to look into this as part of a larger update to XrmFramework :-)

For now, if you do not want to call "Update-Package -reinstall", simply keep the binary folders committed (preferably with LFS).

**Comment by ferodom** [comment]
That makes sense. I commmited the files to Git and it got it working. If I can help with the overhaul please let me know.
